mineinabyss-features/src/main/kotlin/com/mineinabyss/playerprofile/PlayerProfileFeature.kt

```diff
@@ -0,0 +1,41 @@
+package com.mineinabyss.playerprofile
+
+import com.mineinabyss.guiy.inventory.guiy
+import com.mineinabyss.idofront.commands.extensions.actions.playerAction
+import com.mineinabyss.mineinabyss.core.AbyssFeature
+import com.mineinabyss.mineinabyss.core.MineInAbyssPlugin
+import com.mineinabyss.mineinabyss.core.commands
+import org.bukkit.Bukkit
+import org.bukkit.entity.Player
+
+class PlayerProfileFeature : AbyssFeature {
+
+    override fun MineInAbyssPlugin.enableFeature() {
+
+        commands {
+            mineinabyss {
+                "profile"(desc = "Opens a players profile") {
+                    playerAction {
+                        guiy { PlayerProfile(sender as Player, player) }
```

Comment:
Handle error if sender isnt player (ex used from console)

Reply:
playerAction is very misleading as it is. By the sound of it its an action done by the sender as Player
but player is essentially the next argument. Feel like it should essentially just ensure that the sender is player and further arguments should be done inside the cmd?

Reply:
yeah that was meant as a way for console to be able to run player commands as another player

Reply:
it also lets other players run commands as if theyre other players tho
the second argument there doesnt care if sender is a console or player so